server/src/main/java/com/objectcomputing/checkins/services/questions/QuestionServices.java

```diff
@@ -60,11 +52,16 @@ protected Question findByQuestionId(UUID skillId) {
         return skillList;
     }
 
-//    public Question update(Question question) {
-//        Question returned = questionRepository.update(question);
-//
-//        return returned;
-//
-//    }
+    public Question update(Question question) {
+        Question returned = null;
+        Question questionInDatabase = findByQuestionId(question.getQuestionid());
+        if ((questionInDatabase != null)
+                && questionInDatabase.getText().equals(question.getText())) {
```

Comment:
Should this be !equals? This only updates if the text is the same?

Reply:
Good catch! Got it fixed